### ISSUE TITLE
Week 5 TEAM Ticket

### ISSUE THREAD

**Issue by hetvit27** [hetvit27/hetvitrivedi#8]
Hetvi Trivedi: [Link](https://github.com/hetvit27/hetvitrivedi/issues/7) 
Brian Tang:[ Link](https://github.com/bgt072105/super-barnacle/issues/7) 
Jazair Tallman: [Link](https://github.com/jz21324/Vcode/issues/7#issue-1386448146)
Ellen Xu:

**Comment by rjawesome** [comment]
Good idea and brain write which gets interests of the teams. Good wireframe showing the basic design. Doesn't include the abstract classes but has the extends, but even then the code is good and achieves the objectives. The javascript is definbetly good and applies to their project & showing multiple projects. Uses the classes as well in javascript.

I would give a 2.7/3
(feedback from group: Kinish, Jay, Gabe, Rohan)

**Comment by horizon-ab** [comment]
Grade: 2.7+

Project: Interactive physics-dedicated learning website with simulations
- In depth planning
- Detailed wireframe for homepage and simluations

Fibonacci
- Used extends

Javascript
- Coulomb Calculator
- Detailed table of planets and their properties on fastpages
- Calculator
- Team roles table

**Comment by RohanG326** [comment]
Rohan Gaikwad
- project plan was really really detailed and well done
- fibonacci exended and iteration was efficient for the for loop
- same with while, efficient and done in few lines of code
- JS coulomb calculator was well done working in js and html
- the planets table was really done and done using a class and constructor
- each had different js with functionality

**Comment by supermengman** [comment]
Good idea, description and planning or PBL project for Physics EM. 
Fibo extends the main class, no abstract but good work towards it
Ellen: Thorough JS project with an idea for planets and creating a table of the masses/gravity of the planets
- debug on localhost, added into fastpages

Hetvi follow tutorial, with javascript code
Jazair useful JS project for team roles

Overall 2.7+

**Comment by nicm2** [comment]
Good idea and purpose behind PBL project. Well thought out wireframes and future design for project. Lot of effort. Good work with fibo and extending classes. Identifies work that still needs to be done and understands what still needs to be learned. JavaScript project shows purpose and ties back to PBL. Shows use of frontened HTML and use of Javascript. Understanding of Mr. M's Object to table. 

2.7+/3

**Comment by jm1021** [comment]
PBL/Hetvi - Physics topics.  Calculator, Video, and other resources to learning.  Using Crash White as resource.  Made a very nice looking wire frame to break down topics.   

Brian - 90 % of extend is the same as abstract.  Classic algorithms.  2.7
Ellen - Coulomb Calculator adapted for their PBL idea.  Discussed Class for Planets and specific Planets.  Shows understand.  Very nice production of Planet. 2.9
Hetvi - Built a basic calculator.  2.9
Jazair - worked with original JavaScript.  2.7
